scenario/node/src/fabric.ts

```diff
@@ -209,8 +228,8 @@ export class Fabric {
             '--tlsRootCertFiles',
             '/etc/hyperledger/configtx/crypto-config/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt');
 
-        this.runningChaincodes[mangledName] = true;
-        await sleep(10000);
+        this.runningChaincodes[mangledName] = signaturePolicy;
+        await sleep(30000);
```

Comment:
For another day, but we really should find something better than this. Maybe polling the command to list running chaincodes until it appears?

Reply:
I wonder if these sleeps are carried over from the old chaincode lifecycle.  We are now specifying `--waitForEvent` on the commit so hopefully these sleeps aren't needed at all now.  I've taken them out and it works locally, so see what it does in the CI.

Reply:
Oh well, that was a partial success.  Failed in Go - looks like the new endorsement policy hadn't been picked up in the first private-data test.  Adding short sleep back in. Not sure what the `--waitForEvent` is doing then.

Reply:
My _guess_ is that the `--waitForEvent` is waiting for the config update to be committed, but at that point some of the behind-the-scenes deployment work may not yet be complete